stores/searchHealthcareProfessionalStore.ts

```diff
@@ -0,0 +1,53 @@
+import { gql } from 'graphql-request'
+import { defineStore } from 'pinia'
+import { type Ref, ref } from 'vue'
+import { gqlClient } from '../utils/graphql.js'
+import type { HealthcareProfessional } from '~/typedefs/gqlTypes'
+
+export const useSearchHealthcareProfessionalStore = defineStore(
+    'SearchHealthcareProfessionalStore',
+    () => {
+        const healthcareProfessional: Ref<HealthcareProfessional[]> = ref([])
```

Comment:
I don't think we actually need ` getHealthcareProfessional` or `healthcareProfessional`. We can just expose `getHealthcareProfessionalById` and let the caller use the value to set the ref in the component